lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -294,7 +294,7 @@ def attributes_for(serializer, fields)
 
       # {http://jsonapi.org/format/#document-resource-objects Document Resource Objects}
       def resource_object_for(serializer)
-        resource_object = serializer.cache_check(self) do
+        resource_object = serializer.fetch(self) do
```

Comment:
Part of my goal in the direction I've been going with these PRs is to be able to make it easier to pipeline, which caching in second level.